timescaledb/how-to-guides/continuous-aggregates/real-time-aggregates.md

```diff
@@ -28,4 +28,92 @@ You can enable and disable real time aggregation by setting the
 
 </procedure>
 
+## Limitations
+
+Real-time aggregates don't show the data updated in the bucket that was previously
+materialized. The following example illustrates this limitation:
+
+```sql
+-- Create and fill the hypertable
+CREATE TABLE conditions(
+  day DATE NOT NULL,
+  city text NOT NULL,
+  temperature INT NOT NULL);
+
+SELECT create_hypertable(
+  'conditions', 'day',
+  chunk_time_interval => INTERVAL '1 day'
+);
+
+INSERT INTO conditions (day, city, temperature) VALUES
+  ('2021-06-14', 'Moscow', 26),
+  ('2021-06-15', 'Moscow', 22),
+  ('2021-06-16', 'Moscow', 24),
+  ('2021-06-17', 'Moscow', 24),
+  ('2021-06-18', 'Moscow', 27),
+  ('2021-06-19', 'Moscow', 28),
+  ('2021-06-20', 'Moscow', 30),
+  ('2021-06-21', 'Moscow', 31),
+  ('2021-06-22', 'Moscow', 34),
+  ('2021-06-23', 'Moscow', 34),
+  ('2021-06-24', 'Moscow', 34),
+  ('2021-06-25', 'Moscow', 32),
+  ('2021-06-26', 'Moscow', 32),
+  ('2021-06-27', 'Moscow', 31);
```

Comment:
This example does not show the full picture. If you remove `('2021-06-27', 'Moscow', 31)` from conditions and insert it later (instead of just updating old data) it will show the users that even fresh data are not visible by real-time aggregates.

Reply:
The freshness is determined by the timestamp, not by the time WHEN the data was inserted. Feel free to suggest a better wording in the text.

Reply:
It's the timestamp I'm talking about. This example doesn't show that even the freshest timestamp would not be visible.

Reply:
Please take a look at the updated text. I hope it's better now. In this example only one bucket is refreshed, thus if you insert now() it will be visible in the CAGG.